src/parse-row.js

```diff
@@ -1,6 +1,10 @@
 module.exports = function (schema, row, transform) {
   return row.reduce(function (result, cell, index) {
-    result[schema[index]] = transform(cell.v)
+    if (typeof cell === 'object' && cell !== null) {
+      result[schema[index]] = transform(cell.v)
+    } else {
+      result[schema[index]] = null
+    }
```

Comment:
I think that `null` values should also be passed to the `transform` function (for example, we might want to replace all empty cells with some other value). What about doing something like:

```js
result[schema[index]] = transform(typeof cell === 'object' ? cell.v : null)
```